cyfra-runtime/src/main/scala/io/computenode/cyfra/runtime/SpirvDisassembler.scala

```diff
@@ -0,0 +1,106 @@
+package io.computenode.cyfra.runtime
+
+import io.computenode.cyfra.utility.Logger.logger
+
+import java.io.{ByteArrayInputStream, ByteArrayOutputStream}
+import java.nio.ByteBuffer
+import scala.sys.process.*
+
+class SpirvDisassemblerError(msg: String) extends RuntimeException(msg)
```

Comment:
It should be a case class and it would be better to put it in companion object (it's a convention to limit top-level definitions that are named differently than the file)